packages/kit/src/runtime/server/cookie.js

```diff
@@ -1,49 +1,77 @@
-import * as cookie from 'cookie';
+import { parse } from 'cookie';
+
+/** @type {import('cookie').CookieSerializeOptions} */
+const DEFAULT_SERIALIZE_OPTIONS = {
+	httpOnly: true,
+	secure: true,
+	path: '/',
```

Comment:
@cdcarson what's the rationale for having a default `path` value? For `httpOnly` and `secure` the defaults add security, and `sameSite` makes sense because it's already a default, and specifying it explicitly squelches console warnings.

But defaulting `path` to `/` makes things _less_ secure. Even though it's behaviour you often want, should it be the default?

Reply:
Woops -- responded in the other PR: https://github.com/sveltejs/kit/pull/6648#issuecomment-1247237974. Pasted:

@Rich-Harris Pretty sure it is necessary. See https://github.com/sveltejs/kit/issues/6650. Basically a cookie set in a request to /sign-in will have that path. A subsequent request to /profile won't send that cookie.

The only downside of having the default is if the site base path is /foo. We can probably address this in the docs.

Reply:
> As I understand it, we need the `cookies` API because cookies may be written and read in the _same request_, in different functions/files, especially with `actions`

We're not really any worse off because of `cookies`; the same is true if you add a `set-cookie` header any other way. The only difference is that we now have the _opportunity_ to set a default, and it's sorely tempting to take it.

To tl;dr a conversation among the maintainers just now: 👍 to removing the default path. Throwing an error feels heavy-handed and would be a breaking change, but if people _do_ end up getting confused by this stuff then a dev-time warning could be appropriate in future.

Agree re docs, will add something brief to this PR. We needed docs anyway because of the `httpOnly` default.

Reply:
> We're not really any worse off because of cookies; the same is true if you add a set-cookie header any other way.

I guess I was harking back to the old days, doing my own form enhancement. I don't remember having to read a cookie after writing it -- but I was probably doing a whole bunch of goofy inefficient stuff instead.

Agree throwing would be bad. A warning might be nice -- it's an easy mistake.